lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -35,6 +37,9 @@ class EmbedSanitizer
     # Creates a callable transformer for the sanitizer to use
     def self.transformer
       lambda do |env|
+        # Don't continue if this node is already safelisted.
+        return if env[:is_whitelisted]
```

Comment:
Could you explain how something would already be safelisted? (For me, not necessarily in the comment.)

Reply:
Transformers can safelist particular nodes, so they don't get cleaned up by the overall Sanitize config (e.g. we don't allow `<embed>` in `Sanitize::Config::ARCHIVE`, so `EmbedSanitizer` has to safelist such nodes).

If a node has been safelisted by a previous transformer, subsequent transformers ([`MediaSanitizer` in our case](https://github.com/otwcode/otwarchive/blob/0622729c9d4fca1ec7b611235bb5e318256ebd70/lib/html_cleaner.rb#L157-L158)) can know about it by checking [`:is_whitelisted`](https://github.com/rgrove/sanitize/tree/v4.6.6#transformers).

It's not required to honor `:is_whitelisted` (and indeed we were fine not doing so until now), but since `EmbedSanitizer` can now generate nodes that should be sanitized by `MediaSanitizer` (`<audio>`), I'd like to make it clear what is/isn't safelisted and follow that in all of our sanitizers.

Reply:
Thank you! I couldn't find that `is_whitelisted` documentation when I was Googling, and I, uh. Just realized that's because I had the name of the gem wrong. (I was looking for "sanitizer" with an "r".) 🤦

Reply:
I also had to link to the README of the 4.x version we're using, because the option was renamed to `:is_allowlisted` in Sanitize 5.x.